content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade.md

```diff
@@ -141,16 +141,34 @@ Find the latest stable 1.18 version:
         kubeadm upgrade apply v1.18.0
 
     _____________________________________________________________________
+
+    The table below shows the current state of component configs as understood by this version of kubeadm.
+    Configs that have a "yes" mark in the "MANUAL UPGRADE REQUIRED" column require manual config upgrade or
+    resetting to kubeadm defaults before a successful upgrade can be performed. The version to manually
+    upgrade to is denoted in the "PREFERRED VERSION" column.
+
+    API GROUP                 CURRENT VERSION   PREFERRED VERSION   MANUAL UPGRADE REQUIRED
+    kubeproxy.config.k8s.io   v1alpha1          v1alpha1            no
+    kubelet.config.k8s.io     v1beta1           v1beta1             no
+    _____________________________________________________________________
+
     ```
 
     This command checks that your cluster can be upgraded, and fetches the versions you can upgrade to.
+    It also shows a table with the component config version states.
 
 {{< note >}}
 `kubeadm upgrade` also automatically renews the certificates that it manages on this node.
 To opt-out of certificate renewal the flag `--certificate-renewal=false` can be used.
 For more information see the [certificate management guide](/docs/tasks/administer-cluster/kubeadmkubeadm-certs).
 {{</ note >}}
 
+{{< note >}}
+If `kubeadm upgrade plan` shows any component configs that require manual upgrade, users must provide
+a config file with replacement configs to `kubeadm upgrade apply` via the `--config` command line flag.
+Failing to do so will cause `kubeadm upgrade apply` to exit with an error and not perform an upgrade.
```

Comment:
My idea here is for this note to also contain a link at the end to a different page describing in detail the workflow in that case. For that matter, however, we should have the alpha commands from the KEP merged as well (possibly in the next release cycle).

Reply:
> My idea here is for this note to also contain a link at the end to a different page describing in detail the workflow in that case. 

if the new page is going to be at k/website it's fine because the k8s.io docs are branched.
if it links at k/kubeadm, the repo is not branched, so we better have SHAs in the link.

Reply:
I am thinking more about using k/website for that. That's why I want the full set of command line additions to merge before doing the separate page.